common/analysis/processingChainValidator.js

```diff
@@ -51,7 +51,7 @@ export const validateStep = async (step) => {
   )(validation)
 }
 
-export const validateCalculation = async (calculation, defaultLang) =>
+export const validateCalculation = (calculation, defaultLang) =>
```

Comment:
isn't Validator.validate an async function? in that case better to leave async here as well